pkg/ebpf/c/maps.h

```diff
@@ -49,7 +49,7 @@ typedef struct containers_map containers_map_t;
 // persist args between function entry and return
 struct args_map {
     __uint(type, BPF_MAP_TYPE_HASH);
-    __uint(max_entries, 1024);
+    __uint(max_entries, 10240);
```

Comment:
What's the reason for increasing?

Reply:
Since I removed the should_trace() check in TRACE_ENT_FUNC macro, we may have to save more arguments between entry and return (theoretically, assuming filtering would have prevent that save) - so I allocated a bigger map size to mitigate this